### PR TITLE
Log successful calls to uploadData() to synchronizer

### PR DESCRIPTION
**Pull request by ndowmon**

### CODE REVIEW

**Review by mknoedel — APPROVED** [review]

**Comment by ndowmon** [comment]
Also changed project to version together.

File: lerna.json [hunk 1]
```diff
@@ -4,5 +4,5 @@
     "packages/integration-sdk-*",
     "packages/cli"
   ],
-  "version": "independent"
```
Comment (austinkelleher) [inline lerna.json:7]: Can you explain why this change was made?

Comment (ndowmon) [reply lerna.json:7]: Yeah. We don’t independently version this project (at the moment) and the cli command ‘yarn lerna version’ (without adding the version number) makes you go through and version each of the like 6 packages independently. It’s sort of an annoying tedious workflow for me so I would prefer versioning the project together unless/until we actually do choose to version independently.

**Review by austinkelleher — APPROVED** [review]
